src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,12 +6,16 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
 
```

Comment:
great work, python also allows for something like this: arr[cur_index],arr[smallest_index] = arr[smallest_index], arr[cur_index]